### PR TITLE
Write server testing docs

### PR DESCRIPTION
**Pull request by felix920506**
Add some docs on how to test server

### CODE REVIEW

File: docs/general/testing/server.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+The weekly unstable builds are generated every Monday 5:00 AM UTC. Testing using the Unstable builds can not only test Jellyfin itself, but also help identify issues with packaging.
+
+To install them, navigate to our [downloads page](/downloads/server) and choose `Server`, choose your platform, then choose `Unstable`. The choices have to be made in this order. After choosing, scroll down and find the installation method you would like to use.
+
+![Downloads Page](/images/docs/testing/server/weekly-1.png)
+
+## Testing from Master Branch
+
+For the absolute bleeding edge, you can test directly from the Master branch of the server. To test from the master branch, a few extra tools are needed:
```
Comment (joshuaboniface) [inline docs/general/testing/server.md:24]: I'd avoid mentioning this here, or at least, not in this way. Really only *developers* should be building off master like this, not normal users.

File: docs/general/testing/server.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+:::caution Backups!
+
+Unstable often makes irreversible changes to existing Jellyfin setups. Please make regular backups of Jellyfin files. [Read More](/docs/general/administration/configuration#server-paths)
+
+:::
+
+## Weekly Unstable Build
+
+The weekly unstable builds are generated every Monday 5:00 AM UTC. Testing using the Unstable builds can not only test Jellyfin itself, but also help identify issues with packaging.
```
Comment (joshuaboniface) [inline docs/general/testing/server.md:16]: It's actually 5AM EST, or ~1AM UTC (depending on DST).

**Comment by TimGels** [comment]
Where is the description?

File: docs/general/testing/server.md (hunk 1 above)
Comment (felix920506) [reply docs/general/testing/server.md:24]: Maybe move this to a different section of the docs and just leave a link here?

**Comment by felix920506** [comment]
Added description

**Review by joshuaboniface — APPROVED** [review]

**Review by TimGels — APPROVED** [review]

File: docs/general/testing/server.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+uid: testing-server
+title: Testing Jellyfin Server
+---
+
+# Testing Jellyfin Server
+
+:::caution Backups!
```
Comment (nielsvanvelzen) [inline docs/general/testing/server.md:8]: Stop
Adding
Unnecessary
Admonitions

**Pull request merged**